### DISCUSSION TITLE
error saat render plot dengan ggplotly

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#681] [Q&A] [answered]
`transportation_type_proportion_chart <- passenger_num_by_tansportation_type %>% 
  ggplot(mapping = aes(x = "",
                       y = proportion,
                       fill = jenis
                       )
         )+
  geom_col()+
  coord_polar(theta = "y")+
  geom_text(
    aes(label = paste0(round(proportion, 2), "%")),
    position = position_stack(vjust = 0.55)
    )+
  labs(
    title = "Transportation Type Proportion",
    fill = "Transportation Type",
    x = NULL,
    y = NULL
  )+
  theme_void()

transportation_type_proportion_chart`

![image](https://github.com/teamalgoritma/community/assets/23212456/0292e918-2284-43d6-88b9-3500465e190f)

saya sudah melakukan debugging, ternyata error tersebut disebabkan dari `coord_polar`.
![image](https://github.com/teamalgoritma/community/assets/23212456/e3fee7ec-2c7b-4b19-b968-e9fd326ec9b9)

![image](https://github.com/teamalgoritma/community/assets/23212456/8a2ea55e-91da-478d-b750-f9852d4b91f5)

apa peyebab dari error ini?

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @Gustana 
Berdasarkan error yang ada, memang betul diagram pie tidak bisa langsung diubah dari ggplot menjadi ggplotly. Karena ggplotly tidak mendukung diagram pie. Sehingga jika memang ingin melakukan visualisasi diagram pie bisa langsung menggunakan plotly langsung. Tetapi kami sarankan untuk tidak membuat visualisasi dengan data kategorikal bisa menggunakan diagram bar saja dibandingkan dengan pie. 
https://plotly.com/python/pie-charts/

**Reply by Gustana** [reply]
oke, makasi kak
